pkg/sqlcmd/format.go

```diff
@@ -451,7 +461,26 @@ func (f *sqlCmdFormatterType) scanRow(rows *sql.Rows) ([]string, error) {
 				row[n] = x
 			case time.Time:
 				// Go lacks any way to get the user's preferred time format or even the system default

```

Comment:
If I am reading this correctly, there is now a way to get the format string, yes?  Can we strike this comment now?

Reply:
No, I simply used the same format SQL server uses by default, modified by the precision value for the number of decimal places.
I opened #80 to discuss ways to allow user-provided format strings.